light/client_test.go

```diff
@@ -206,9 +205,15 @@ func TestClient_SequentialVerification(t *testing.T) {
 		},
 	}
 
+	bctx, bcancel := context.WithCancel(context.Background())
```

Comment:
Could we do this inside the test case? I don't think we need a "root" context here (unless I missed something).

Reply:
yeah, I just did this by default